clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -741,6 +756,9 @@ public void onResponse(final ClientResponse response) {
                             "failed with unknown member ID. " + error.message()));
                         return;
                     } else if (error == Errors.STALE_MEMBER_EPOCH) {
+                        log.error("OffsetCommit failed for member {} with stale member epoch error. Last epoch sent: {}",
+                            memberInfo.memberId.orElse("undefined"),
+                            lastEpochSentOnCommit.isPresent() ? lastEpochSentOnCommit.get() : "None");
```

Comment:
maybe we can replace "None" by "undefined" for consistency?

Reply:
Sure, done.